src/Rings/MPolyMap/base_change.jl

```diff
@@ -0,0 +1,29 @@
+function change_base_ring(A::Ring, P::MPolyRing)
+  kk = coefficient_ring(P)
+  res, _ = polynomial_ring(A, symbols(P), cached=false)
+  return res, hom(P, res, A, gens(res), check=false)
+end
+
+function change_base_ring(A::Ring, Q::MPolyQuoRing)
+  P = base_ring(Q)
+  PA, phi = change_base_ring(A, P)
+  I = phi(modulus(Q))
+  res, pr = quo(PA, I)
+  return res, hom(Q, res, A, gens(res), check=false)
+end
```

Comment:
where do you use 'pr'?

Reply:
It's not used. But this is not a problem.